backend/app/api/v1/llm.py

```
@@ -11,48 +11,37 @@
-        id_report = await report_service.save_report(
-            db=db,
-            measurements=measurements_dict,
-            photo_path=photo_path,
-            meta=meta,
-            llm_response=llm_response,
-            trace_data=trace_data
-        )
+        id_report = await report_service.save_report(db=db, measurements=measurements_dict, photo_path=photo_path,
+                                                     meta=meta, llm_response=llm_response.get("report"), trace_data=trace_data)
         return ReportIdResponse(id=id_report)
     except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)